runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -2914,7 +2914,7 @@ else if (this.authField.equals(MqttConnectProperty.PASSWORD))
                     if (credentialsMatch != null)
                     {
                         sessionAuth = guard.reauthorize(initialId, credentialsMatch);
-                        event.authorization(sessionAuth, traceId, routedId, guard.identity(sessionId));
+                        event.authorizationFailure(sessionAuth, traceId, routedId, guard, sessionId);
```

Comment:
Check for failure, call conditionally, remove parameters not used. We need a way to get attempted authorization identity even when authorization fails.